test/compiler/inline.jl

```diff
@@ -672,3 +672,7 @@ end
         success(pipeline(Cmd(cmd); stdout=stdout, stderr=stderr))
     end
 end
+
+# Issue #42264 - crash on certain union splits
+f(x) = (x...,)
```

Comment:
```suggestion
let f(x) = (x...,)
```
to avoid namespace pollution